Source/WebCore/dom/Node.h

```diff
@@ -784,8 +777,6 @@ class Node : public EventTarget, public CanMakeCheckedPtr<Node> {
     mutable bool m_inRemovedLastRefFunction { false };
     bool m_adoptionIsRequired { true };
     bool m_deletionHasBegun { false };
-
-    friend void adopted(Node*);
```

Comment:
This seems to have gotten dropped and I don't quite understand why.

Reply:
Turns out this was not unused after all :( Debug builds rely on this to update `m_adoptionIsRequired`. Resurrected the function.